server/routes/utils/reportHelper.ts

```diff
@@ -70,26 +73,30 @@ export const generatePDF = async (
       headless: true,
     });
     const page = await browser.newPage();
-    await page.goto(url, { waitUntil: 'networkidle0' });
 
+    await page.setDefaultNavigationTimeout(0);
```

Comment:
What's this option for

Reply:
Prevent a timeout error occurring if the report takes longer to generate than 30 seconds